src/pyodide/internal/tar.js

```diff
@@ -0,0 +1,83 @@
+import { default as Reader } from "pyodide-internal:packages_tar_reader";
+
```

Comment:
Is the code in this file based on any libraries/specs we could link to for future reference?

Reply:
Also it may be worth writing about the limitations of this tar parser (if there are any).

Reply:
I think I just read the wikipedia page on the tar format:
https://en.wikipedia.org/wiki/Tar_(computing)#File_format
I will add a comment with a link to it.